src/hyperdeck.ts

```diff
@@ -126,7 +126,7 @@ export class Hyperdeck extends EventEmitter<HyperdeckEvents> {
 	}
 
 	async sendCommand<TResponse>(command: AbstractCommand<TResponse>): Promise<TResponse> {
-		if (!this._connected) return Promise.reject()
+		if (!this._connected) return Promise.reject(new Error('Hyperdeck not connected'))
```

Comment:
```suggestion
		if (!this._connected) throw new Error('Hyperdeck not connected')
```
Could also be written like this